Dockerfiles/Dockerfile.latest

```diff
@@ -47,6 +47,10 @@ LABEL \
 	maintainer="cytopia <cytopia@everythingcli.org>" \
 	repo="https://github.com/cytopia/docker-eslint"
 COPY --from=builder /usr/local/lib/node_modules/ /usr/local/lib/node_modules/
+
+# Move the sarif reporting module to the formatters location
+RUN mv /usr/local/lib/node_modules/@microsoft /usr/local/lib/node_modules/eslint/lib/cli-engine/formatters
```

Comment:
This should be in the form of 
```
RUN set -eux \
	&& mv /usr/local/lib/node_modules/@microsoft /usr/local/lib/node_modules/eslint/lib/cli-engine/formatters
```

Reply:
You're right, to keep in conformity with other commands, updated the file to resolve the problem.